authority/provisioner/wire/oidc_options.go

```diff
@@ -62,6 +66,8 @@ func (o *OIDCOptions) GetConfig() *oidc.Config {
 	}
 }
 
+const defaultTemplate = `{"name": "{{ .name }}", "handle": "{{ .preferred_username }}"}`
```

Comment:
Should we care about quotes in those variables?

Something like this will take care of it:
```
const defaultTemplate =  `{{ printf "{\"name\": %q, \"handle\": %q}"  .name .preferred_username }}`
```